specs/global_navigation.adoc

```diff
@@ -0,0 +1,74 @@
+# Initial UI refresh for the grafana integration
+
+With the integration of grafana as the monitoring dashboard, the functionality offered by the tendrl ui is being refined. This specification talks about the functionality retained from the last ui build (available as the [ui-legacy-code](https://github.com/Tendrl/ui/tree/ui-legacy-code) branch) and the resulting changes to the global navigation.
+
+## Proposed change
+
+Currently the navigation has following items:-
+
+* Dashboard
+* Clusters
+* Hosts
+* Volumes
+* Pools
+* RBDs
+* Alerts
+* Admin
+
+
+The following changes has been done in the navigation
+
+* Removing *Cluster specific objects* items such as *Volumes*, *Pools* and *Rbds*.
+* Retaining items like *Clusters*, *Hosts*, *Admin(tasks)*, *Alerts*
+
+** Cluster List ```module: cluster```
+** Hosts list ```module: host```
+** Global alerts list ```module: alerts```
+** Tasks list ```module: tasks```
+** Task details page ```module: tasks/task-detail```
```

Comment:
Why is the task detail page on the main navigation? Shouldn't that be accessible only via the Tasks list page?